src/components/navBar/demoBasicUsage/index.html

```diff
@@ -13,3 +14,11 @@
       External content for `<span>{{currentNavItem}}</span>`
     </div>
   </md-content>
+
+  <div class="md-padding" layout="row" layout-sm="column" layout-align="left center"
+       style="padding-top: 0;">
```

Comment:
Please remove the `layout` and `layout-align` here (we don't need the alignment and a layout for now)

![](https://i.gyazo.com/16dc9e81e5de79661c70940430caf330.png)